src/Raven.Client/Documents/Session/AbstractDocumentQuery.cs

```diff
@@ -1143,6 +1180,30 @@ private void BuildInclude(StringBuilder queryText)
                 }
             }
 
+            if (CounterIncludes != null)
+            {
+                if (first == false)
+                    queryText.Append(",");
+
+                queryText.Append("counters(");
```

Comment:
I'm wondering if we should not support parameters here? @ayende ?

Reply:
And yes, we want to handle this via parameters, not direct injection of string into the query

Reply:
done